src/ontology/exo.obo

```diff
@@ -1557,8 +1557,9 @@ xref: BFO:0000051
 is_transitive: true
 
 [Typedef]
-id: interacts_with
-name: interacts_with
+id: interacts with
```

Comment:
`id` should probably be `interacts_with` (the `name` is correct).

The rest looks good.

Once done you should be able to use ROBOT to save the ontology as exo.owl on the top level of the repo as well as (again) as a exo.obo so that all three files are updated.